hooks/useTokenList.tsx

```diff
@@ -1,6 +1,6 @@
 import { ethAddress } from "@/utils/constants";
 import { networks } from "@/utils/networks";
-import useSWR from "swr";
+import useSWRImmutable from "swr/immutable";
```

Comment:
_(arbitrary comment location to start a thread)_

Seeing these errors in the console, but I was able to shield funds still. Showed up on both mainnet and goerli, haven't tried other networks

![image](https://user-images.githubusercontent.com/17163988/214387440-3718bc4a-6ad5-4289-b3ea-8e923e482232.png)

Reply:
I think this is coming from the loading the ArbitrumGoerli network. Going to remove and we can add back

Reply:
#64